### PR TITLE
Some updates to the documentation

### PR DESCRIPTION
**Pull request by ItzLevvie**
***This PR adds the following***:
1. Updated all TOCs to match the latest changes from [here](https://github.com/MicrosoftDocs/windows-insider/commit/3eed128b56e4010780a86c9d864ea2a42b09db80#diff-32b4b31d046c4ed937d759734d2c527b)

2. Added a hyperlink under the ```Windows Snip``` subheader in the ```wip/at-home/Whats-new-wip-at-home-1903.md``` file to match official blog post from [here](https://blogs.windows.com/windowsexperience/2018/11/14/announcing-windows-10-insider-preview-build-18282/)

    - Before:
    image
    - After:
    image

***This PR fixes the following***:
1. Fixed a Wikipedia link not redirecting to the ```Japanese style (kaomoji)``` header when clicked

2. Fixed the shrug (¯\\\_(ツ)_/¯) kaomoji not displaying correctly from a piece of content from the ```Easy Symbols and Kaomoji Access``` subheader under the ```Notable New Features``` header in the ```wip/at-home/Whats-new-wip-at-home-1903.md``` file

    - Before:
    image
    - After:
    image

3. Removed unneeded asterisk (*) from a piece of content under the ```Emoji 11``` header in the ```wip/at-home/Whats-new-wip-at-home-1809.md``` file

    - Before:
    image
    - After:
    image

4. Removed unneeded asterisks (**) from the ```Send Feedback``` subheader under the ```Notepad Updates``` header in the ```wip/at-home/Whats-new-wip-at-home-1903.md``` file

    - Before:
    image
    - After:
    image

5. Removed unneeded asterisk (*) from a piece of content under the ```Introducing a new Cortana experience for Windows 10 (Build 18945)``` header in the ```wip/at-home/Whats-new-wip-at-home-20h1.md``` file

    - Before:
    image
    - After:
    image

6. Removed unneeded underscore (_) from a piece of content from the ```Photos has evolved. View and interact with your most recent 2000 photos!``` subheader under the ```Your Phone``` header in the ```wip/at-home/whats-new-apps.md``` file

    - Before:
    image
    - After:
    image

7. Added an additional underscore (_) to a piece of content from the ```PowerToys v0.12 now available!``` subheader under the ```PowerToys``` header in the ```wip/at-home/whats-new-apps.md``` file

    - Before:
    image
    - After:
    image

8. Removed unneeded asterisks (**) from a piece of content under the ```Sticky Notes``` header in the ```wip/at-home/whats-new-apps.md``` file

    - Before:
    image
    - After:
    image

***This PR fixes the following sentence errors***:
1. Changed ```supports**Ctrl + Backspace**``` to ```supports **Ctrl + Backspace**```

    - Before:
    image
    - After:
    image

***This PR fixes the following spelling mistakes***:
1. Changed ```showibg``` to ```showing```

    - Before:
    image
    - After:
    image

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/at-home/Whats-new-wip-at-home-1903.md [hunk 1]
```diff
@@ -95,12 +95,12 @@ Have you already tried using clipboard history? If enabled, it allows you to sav
...
 Currently only the emoji section supports type to search.
 
-This experience, originally introduced for Chinese (Simplified) during RS5 development, started rolling out to Insiders across all languages with [Build 18252](https://blogs.windows.com/windowsexperience/2018/10/03/announcing-windows-10-insider-preview-build-18252/) – as of today’s it’s now available to all Insiders in Fast! We’d love for you to try it out and share feedback. Tried it out already? With this build, we’ve updated the available kaomoji a bit based on your feedback, including adding ¯\_(ツ)_/¯ and ᓚᘏᗢ.
+This experience, originally introduced for Chinese (Simplified) during RS5 development, started rolling out to Insiders across all languages with [Build 18252](https://blogs.windows.com/windowsexperience/2018/10/03/announcing-windows-10-insider-preview-build-18252/) – as of today’s it’s now available to all Insiders in Fast! We’d love for you to try it out and share feedback. Tried it out already? With this build, we’ve updated the available kaomoji a bit based on your feedback, including adding ¯\\\_(ツ)_/¯ and ᓚᘏᗢ.
```
Comment (JohanFreelancer9) [inline wip/at-home/Whats-new-wip-at-home-1903.md]:
```suggestion
This experience, originally introduced for Chinese (Simplified) during RS5 development, started rolling out to Insiders across all languages with [Build 18252](https://blogs.windows.com/windowsexperience/2018/10/03/announcing-windows-10-insider-preview-build-18252/) – as of today it’s available to all Insiders in Fast! We’d love for you to try it out and share feedback. Tried it out already? With this build, we’ve updated the available kaomoji a bit based on your feedback, including adding ¯\\\_(ツ)_/¯ and ᓚᘏᗢ.
```

File: wip/at-home/Whats-new-wip-at-home-1903.md [hunk 2]
```diff
@@ -288,7 +288,7 @@ Ultimately, we want to help you make the right decision for your situation. By c
 ![Redesigned sign-in options](images/18272-image1.png)
 
 ### Set up a security key from sign-in settings
-In 19H1, ](#sign-in-options-redesigned-for-windows-hello), we set out to simplify sign-in settings, to increase visual clarity, and improve ease of use. Our goal is to give you a one-stop shop for all your sign-in settings, and today we’re adding one more to the list: You can now set up a security key straight from Settings.
+In 19H1, we set out to simplify sign-in settings, to increase visual clarity, and improve ease of use. Our goal is to give you a one-stop shop for all your sign-in settings, and today we’re adding one more to the list: You can now set up a security key straight from Settings.
```
Comment (JohanFreelancer9) [inline wip/at-home/Whats-new-wip-at-home-1903.md]:
```suggestion
In 19H1, we set out to simplify sign-in settings to increase visual clarity and improve ease of use. Our goal is to give you a one-stop shop for all your sign-in settings, and today we’re adding one more to the list: You can now set up a security key straight from Settings.
```

File: wip/at-home/Whats-new-wip-at-home-20h1.md [hunk 3]
```diff
@@ -382,7 +382,7 @@ It supports most of the Cortana features people enjoy using on Windows, such as
 
 Not all the features from the previous Cortana experience are available just yet. As a Beta, we plan to add more features over time with updates to Cortana from the Microsoft Store.
 
-To get started, choose the Cortana icon on the taskbar next to the search box. You may need to sign-in with your account to get started. You can also leverage the speed and convenience of voice with improved speech recognition by simply saying “Hey Cortana”*. To enable this functionality go to Settings>Privacy>Voice activation> and enable "Let Cortana respond to 'Hey Cortana' keyword".
+To get started, choose the Cortana icon on the taskbar next to the search box. You may need to sign-in with your account to get started. You can also leverage the speed and convenience of voice with improved speech recognition by simply saying “Hey Cortana”. To enable this functionality go to Settings>Privacy>Voice activation> and enable "Let Cortana respond to 'Hey Cortana' keyword".
```
Comment (JohanFreelancer9) [inline wip/at-home/Whats-new-wip-at-home-20h1.md]:
```suggestion
To get started, choose the Cortana icon on the taskbar next to the search box. You may need to sign in with your account to get started. You can also leverage the speed and convenience of voice with improved speech recognition by simply saying “Hey Cortana”. To enable this functionality, go to Setting s> Privacy > Voice activation and enable "Let Cortana respond to 'Hey Cortana' keyword".
```

Comment (mypil) [reply wip/at-home/Whats-new-wip-at-home-20h1.md]: @JohanFreelancer9 - there's an extra space in Settings on this copy/edit. Can you still make the changes for this?

Comment (JohanFreelancer9) [reply wip/at-home/Whats-new-wip-at-home-20h1.md]: @mypil  Thanks for picking that up. Done.

**Comment by mypil** [comment]
@ItzLevvie - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@bleblancMSFT - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**